treegp/two_pcf.py

```diff
@@ -92,15 +92,15 @@ def _model_skl(self, sigma, corr_length, g1, g2):
         from sklearn kernel.
 
         :param sigma:         Standard deviation of the gaussian random field.
-        :param corr_length:   Correlation lenght of the kernel.
+        :param corr_length:   Correlation length of the kernel.
         :param g1, g2:        Shear applied to isotropic kernel.
         """
-        if abs(g1)>1 or abs(g2)>1:
+        if (g1**2 + g2**2)>1:
             return None
         else:
             L = get_correlation_length_matrix(corr_length, g1, g2)
             invLam = np.linalg.inv(L)
-            kernel_used = sigma**2 * self.kernel_class(invLam=invLam)
```

Comment:
Having an example of this failing can be nice. I am not able to reproduce this error.